src/main/asciidoc/java/index.adoc

```diff
@@ -254,9 +246,6 @@ Then override the default JGroups config so that `KUBE_PING` becomes the discove
   <MFC max_credits="2m"
 *        min_threshold="0.40"
 *   />
-  <FRAG2/>
```

Comment:
Guessing you didn't mean to remove FRAG2 here?  Also you can use FRAG3 with JGroups 4 now.